app/uk/gov/hmrc/servicedependencies/connector/model/TeamAndRepository.scala

```diff
@@ -20,7 +20,7 @@ import org.joda.time.DateTime
 import play.api.libs.json.Json
 
 case class GithubInstance(name: String, displayName: String)
```

Comment:
this class is potentially not needed if there is only one github

Reply:
Removed in https://github.com/hmrc/service-dependencies/pull/10/commits/6f38a2c1dc1d22232d9e38d13700acda93bf48b3